### PR TITLE
Remove pre-release annotation from some of the modules

### PR DESCRIPTION
**Pull request by rnjtranjan**

### CODE REVIEW

File: src/Streamly/Internal/FileSystem/Event.hs [hunk 1]
```diff
@@ -3,7 +3,6 @@
 -- Copyright   : (c) 2020 Composewell Technologies
 -- License     : BSD-3-Clause
 -- Maintainer  : streamly@composewell.com
--- Stability   : pre-release
```
Comment (harendra-kumar) [inline src/Streamly/Internal/FileSystem/Event.hs:6]: This should remain pre-release.

File: src/Streamly/Internal/FileSystem/Event/Darwin.hs [hunk 2]
```diff
@@ -3,7 +3,6 @@
 -- Copyright   : (c) 2020 Composewell Technologies
 -- License     : BSD-3-Clause
 -- Maintainer  : streamly@composewell.com
--- Stability   : pre-release
```
Comment (harendra-kumar) [inline src/Streamly/Internal/FileSystem/Event/Darwin.hs:6]: This should remain pre-release.

File: src/Streamly/Internal/FileSystem/Event/Windows.hs [hunk 3]
```diff
@@ -7,7 +7,6 @@
 --               (c) 2012, Mark Dittmer
 -- License     : BSD-3-Clause
 -- Maintainer  : streamly@composewell.com
--- Stability   : pre-release
```
Comment (harendra-kumar) [inline src/Streamly/Internal/FileSystem/Event/Windows.hs:10]: This should remain pre-release.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]

**Comment by rnjtranjan** [comment]
Fixes: https://github.com/composewell/streamly/issues/2409

**Pull request merged**